e2etest/get_stats_test.go

```diff
@@ -48,27 +48,39 @@ func TestGetStats(t *testing.T) {
 	for _, c := range []struct {
 		statsVar     string
 		place        []string
+		mmethod      string
```

Comment:
have a unit test?

Reply:
Done

Reply:
Oh nevermind, later realized the unit isn't really implemented yet (blocked on cache).